connectors/google/package.json

```diff
@@ -3,7 +3,7 @@
     "node": "10"
   },
   "dependencies": {
-      "superagent": "^5.2.2",
-      "@fusebit/add-on-sdk": "^0.0.1"
+    "superagent": "^5.2.2",
+    "@fusebit/add-on-sdk": "^0.0.1"
```

Comment:
Newer SDK?

Reply:
Won't fix, not touching the other addons, just prettifying.